src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ProjectCapability.cs

```diff
@@ -11,12 +11,12 @@ internal static class ProjectCapability
         public const string CSharp = ProjectCapabilities.CSharp;
         public const string VisualBasic = ProjectCapabilities.VB;
         public const string FSharp = nameof(FSharp);
-        public const string Managed = ProjectCapabilities.Managed;
+        public const string CSharpOrVBOrFSharp = "(" + ProjectCapabilities.CSharp + " | " + ProjectCapabilities.VB + " | " + FSharp + ")";
```

Comment:
We call it VisualBasic elsewhere in this class, we should at least be consistent through it - whether that matches "VB" or "VisualBasic".